DSharpPlus.Tests/Commands/Cases/TestSingleLevelSubCommands.cs

```diff
@@ -0,0 +1,23 @@
+namespace DSharpPlus.Tests.Commands.Cases;
+
+using System.Threading.Tasks;
+using DSharpPlus.Commands;
+using DSharpPlus.Commands.Processors.TextCommands;
+using DSharpPlus.Commands.Processors.TextCommands.Attributes;
+using DSharpPlus.Commands.Trees.Attributes;
```

Comment:
as much as i appreciate the sentiment, our code style is putting the usings first

Reply:
I think that's worse than the current Java style namespace declarations. Sure I'll do it

Reply:
Then don't do it :cat_scream:

Reply:
Would we have this issue with block scoped namespaces?

Reply:
yes.